shared/src/simulation/behaviors/report.ts

```diff
@@ -39,6 +42,14 @@ export class ReportBehaviorState implements SimulationBehaviorState {
     @IsBoolean()
     public readonly reportTreatmentProgressChanges: boolean = true;
 
+    @IsBoolean()
+    public readonly reportTransferOfCategoryInSingleRegionCompleted: boolean =
+        true;
+
+    @IsBoolean()
+    public readonly reportTransferOfCategoryInMultipleRegionsCompleted: boolean =
+        true;
+
```

Comment:
In the PA template, could you please set this value to `false` so simple patient trays do not send duplicate radiograms by default?

Reply:
A PA template should not have the manage transports behavior at all, so this should not be a problem.

What I could (and should, if I think about this) do is to disable the single region report per default on all simulated regions, since even a self-managed region would need a management behavior (that would be assigned to the region itself and only manage this region), so the single and multiple region reports would almost be equivalent (despite from the text "this region"/"all managed regions"), but at least some radiograms would be created.

Reply:
I've changed the default for the single region to `false` in 3735008f.